.github/workflows/CI.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
```

Comment:
```suggestion
name: CI for local yaml-parser-action
```

Reply:
@hectorsector we may end up with more CI than just this, so I'm shooting for being explicit here